### PR TITLE
update(ejs): `beautify` option from 3.0.2

### PR DESCRIPTION
**Pull request by peterblazejewicz**
- add missing property with doc comment
- tests updated

https://github.com/mde/ejs/compare/v3.0.1...v3.0.2
https://github.com/mde/ejs#options

Thanks!

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/mde/ejs/compare/v3.0.1...v3.0.2
https://github.com/mde/ejs#options

### CODE REVIEW

File: types/ejs/index.d.ts [hunk 1]
```diff
@@ -408,7 +408,14 @@ export interface Options {
...
 	 */
-	async?: boolean;
+    async?: boolean;
+
+    /**
+     * Make sure to set this to 'false' in order to skip UglifyJS parsing,
+     * when using ES6 features (`const`, etc) as UglifyJS doesn't understand them.
+     * @default true
+     */
+    beautify?: boolean;
```
Comment (ExE-Boss) [inline types/ejs/index.d.ts]:
```suggestion
	async?: boolean;

	/**
	 * Make sure to set this to 'false' in order to skip UglifyJS parsing,
	 * when using ES6 features (`const`, etc) as UglifyJS doesn't understand them.
	 * @default true
	 */
	beautify?: boolean;
```

File: types/ejs/test/ejs.umd.test.ts [hunk 2]
```diff
@@ -36,3 +36,9 @@ expectType<{
 	<T>(path: string, data: ejs.Data, opts: ejs.Options, cb: ejs.RenderFileCallback<T>): T;
 	(path: string, data?: ejs.Data, opts?: ejs.Options): Promise<string>;
 }>(ejs.renderFile);
+
+// https://github.com/mde/ejs#options
+const renderOptions: ejs.Options = {
+    beautify: true,
+    filename: './index.ejs',
+};
```
Comment (ExE-Boss) [inline types/ejs/test/ejs.umd.test.ts]:
```suggestion
const renderOptions: ejs.Options = {
	beautify: true,
	filename: './index.ejs',
};
```

**Review by ExE-Boss — CHANGES_REQUESTED** [review]
This package uses tabs.

Also, the options change should be added to `test/ejs.cjs.test.ts`.

**Comment by peterblazejewicz** [comment]
@ExE-Boss 

> This package uses tabs.

Just recalled the discussion we had, sorry!

**Comment by peterblazejewicz** [comment]
> Also, the options change should be added to `test/ejs.cjs.test.ts`.

tbd, thanks!

File: types/ejs/test/ejs.cjs.test.ts [hunk 3]
```diff
@@ -72,3 +72,9 @@ ejs.cache = LRU(100);
 /** @see https://github.com/mde/ejs/tree/v2.5.7#custom-delimiters */
 ejs.delimiter = '%';
 delete ejs.delimiter;
+
+// https://github.com/mde/ejs#options
+const renderOptions: ejs.Options = {
+    beautify: true,
+    filename: fileName,
+};
```
Comment (ExE-Boss) [inline types/ejs/test/ejs.cjs.test.ts:80]: This also needs to use tabs:

```suggestion
const renderOptions: ejs.Options = {
	beautify: true,
	filename: fileName,
};
```

**Review by ExE-Boss — CHANGES_REQUESTED** [review]

**Review by ExE-Boss — APPROVED** [review]

**Pull request merged**